### ISSUE TITLE
trackpad switch(F9) break.

### ISSUE THREAD

**Issue by paranoiagu** [johnnync13/Xiaomi-Mi-Air#127]
i use latest opencore bootloader.
my trackepad switch is break.

**Comment by johnnync13** [comment]
what do you want to say?

**Comment by paranoiagu** [comment]
> what do you want to say?

i can not shutdown the trackpad when i  press the trackpad switch key(F9).

**Comment by Mikes0** [comment]
Same issue, F9 normally toggles trackpad on/off.  F9 does nothing now.

**Comment by yllwfsh** [comment]
@paranoiagu @Mikes0 I never use this feature to be honest, since I'm used to real macs that don't have this functionality. At times I've hit F9 by accident and pulled my hair out trying to find out why the trackpad didn't work anymore ;)

If you want, there's a way to disable the trackpad when you have an external mouse attached: https://osxtips.net/how-to-disable-trackpad-on-a-macbook/

Having said that, it is interesting that this changed. @johnnync13 Any idea where/when this has been changed? I've tried to debug but ran out of time. Is it related to ACPI or to VoodooPS2?
